spec/models/importers/organisation_spec.rb

```diff
@@ -1,19 +1,40 @@
 require 'rails_helper'
```

Comment:
I would like to see a commit where you add your test, and then another commit where you do the Dry-up. Something similar to what you did in 4dcd67a2e3237bb53cb030a8ad06a29fbe9ebb79